pifpaf/drivers/__init__.py

```diff
@@ -74,7 +74,7 @@ def _exec(self, command, stdout=False, ignore_failure=False,
         complete_env = {}
         app = command[0]
 
-        if stdout or wait_for_line:
+        if stdout or wait_for_line or LOG.getEffectiveLevel() >= logging.DEBUG:
```

Comment:
If I patch without this __init__ file, it will print stack trace after tox -epy27

if I patch this file along with the __main__file, it will stuck when run tox -epy27, I have to CTRL+C to stop it after dozens of minutes.... 

I don't know why :(